ui/app/templates/components/form-field.hbs

```diff
@@ -88,6 +88,12 @@
     inputValue=(get model valuePath)
     onChange=(action (action "setAndBroadcast" valuePath))
   }}
+{{else if (eq attr.options.sensitive true)}}
+  {{masked-input
```

Comment:
I know most of the others are `{{` components now, but I think it'd be good to try and keep new additions to the angle-bracket form.